modules/tracker/controllers/components/ApiComponent.php

```diff
@@ -113,22 +111,12 @@ public function itemAssociate($args)
      * @param communityId The id of the community that owns the producer
```

Comment:
Please add to the top level function doc that a new submission will be created with the passed in uuid if a matching one doesn't exist.

I traced through the code and saw that this was happening, and it appears like correct behavior to me, but was a bit of a surprise.